consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/client/converter/XmlToObjectTest.java

```diff
@@ -56,4 +63,73 @@ void shouldConvertBreathingSpaceRequestAndConvertToJson() throws IOException {
         assertEquals(JSON_RESPONSE, jsonString);
     }
+
+    @Test
+    void shouldConvertJudgementRequestToString() throws IOException {
+        String xmlContent = readXmlAsString(JUDGEMENT);
+        String request = xmlToObject.convertXmlToJson(xmlContent);
+        assertNotNull(request);
+        String expectedContent = readFile(JUDGEMENT_JSON);
+        assertEquals(expectedContent, request);
+    }
+
+    @Test
+    void shouldConvertJudgementRequest() throws IOException {
+        String xmlContent = readXmlAsString(JUDGEMENT);
+        JudgementRequest request = xmlToObject.convertXmlToObject(xmlContent, JudgementRequest.class);
+        assertNotNull(request);
+        assertEquals("9QZ00005", request.getCaseManRef());
+        assertEquals(JudgmentType.A, request.getJudgmentType());
+        assertEquals(true, request.isSentParticularsSeparately());
+        assertEquals(false, request.isJointJudgment());
+        assertNotNull(request.getRespondent1Address());
+        assertEquals("defendant1Address line1", request.getRespondent1Address().getAddressLine1());
+        assertEquals("defendant1Address line2", request.getRespondent1Address().getAddressLine2());
+        assertEquals("defendant1Address line3", request.getRespondent1Address().getAddressLine3());
+        assertEquals("defendant1Address line4", request.getRespondent1Address().getAddressLine4());
+        assertEquals("RG42 2DL", request.getRespondent1Address().getPostCode());
+        assertNull(request.getRespondent1Address().getPostTown());
+        assertNotNull(request.getRespondent1DOB());
+        assertNull(request.getRespondent2Address());
+        assertNull(request.getRespondent2DOB());
+        assertNotNull(request.getPaymentSchedule());
+        assertNull(request.getPaymentSchedule().getPaymentInFullBy());
+        assertNull(request.getPaymentSchedule().getInstallmentAmount());
+        assertNotNull(request.getPaymentSchedule().getImmediatePayment());
+        assertNull(request.getPaymentSchedule().getInstallmentFrequency());
+        assertEquals(99999, request.getInterest());
+        assertEquals(5000, request.getSolicitorCost());
+        assertEquals(10000000, request.getDeductedAmount());
+        assertEquals(null, request.getClaimAmountAdmitted());
+        assertEquals(null, request.getCourtFee());
+        assertEquals(null, request.getLegalCosts());
+        assertNotNull(request.getPayee());
+        assertEquals("payee name", request.getPayee().getName());
+        assertNotNull(request.getPayee().getAddress());
+        assertEquals("payee line1", request.getPayee().getAddress().getAddressLine1());
+        assertEquals("payee line2", request.getPayee().getAddress().getAddressLine2());
+        assertEquals("payee line3", request.getPayee().getAddress().getAddressLine3());
+        assertEquals("payee line4", request.getPayee().getAddress().getAddressLine4());
+        assertEquals(null, request.getPayee().getAddress().getPostTown());
```

Comment:
Could use assertNull() here instead of assertEquals().

Reply:
done